hera_qm/xrfi.py

```diff
@@ -11,64 +12,36 @@ def medmin(d):
     Returns:
         (array): array with the statistic applied.
     """
-    #return np.median(np.min(chisq,axis=0))
-    mn = np.min(d,axis=0)
-    return 2*np.median(mn) - np.min(mn)
+    # return np.median(np.min(chisq,axis=0))
+    mn = np.min(d, axis=0)
+    return 2 * np.median(mn) - np.min(mn)
+
 
 def medminfilt(d, K=8):
     """Filter an array on scales of K indexes with medmin.
-    
+
     Args:
         d (array): 2D data array.
         K (int, optional): integer representing box size to apply statistic.
-    
+
     Returns:
         array: filtered array. Same shape as input array.
     """
     d_sm = np.empty_like(d)
     for i in xrange(d.shape[0]):
         for j in xrange(d.shape[1]):
-            i0,j0 = max(0,i-K), max(0,j-K)
-            i1,j1 = min(d.shape[0], i+K), min(d.shape[1], j+K)
-            d_sm[i,j] = medmin(d[i0:i1,j0:j1])
+            i0, j0 = max(0, i - K), max(0, j - K)
+            i1, j1 = min(d.shape[0], i + K), min(d.shape[1], j + K)
+            d_sm[i, j] = medmin(d[i0:i1, j0:j1])
     return d_sm
 
-#def omni_chisq_to_flags(chisq, K=8, sigma=6, sigl=2):
```

Comment:
Does this code exist elsewhere? Is it not useful?
If there's a chance we will want to keep it, we should move it to a cal_metrics.py file in the dirty_code directory to be reviewed.